### PR TITLE
Logfile

### PR DESCRIPTION
**Pull request by nshoang0808**
Create log file, update, and view when choose "Logs"

### CODE REVIEW

**Comment by nshoang0808** [comment]
I have changed and make sure all function works.

File: routes/index.js [hunk 1]
```diff
@@ -88,4 +89,14 @@ function checkLogin(req,res,next){
...
+router.get('/Logs', function(req, res) {
+  fs.readFile('log.txt', (err, result) => {
+    if (err) {
+    } else {
+			var array = result.toString().split("\n");
+      res.send(array.slice(0, array.length-1));
+    }
+  });
+});
+
```
Comment (ProjectZed) [inline routes/index.js]: Make this into a function, just like you did with writeToLog, make a function readLog that returns the string

File: routes/writeLog.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+var fs = require('fs');
+var express = require('express');
+var sqlite3 = require('sqlite3').verbose();
+var router = express.Router();
+var log = require('log4js').getLogger("index");
+var db = new sqlite3.Database(__dirname + "/../server/database/LibertyMutual.db");
+
+module.exports = {
+  writeToLog: function(data) {
```
Comment (ProjectZed) [inline routes/writeLog.js]: This is where I would add the read log function and rename this file to something like LogInterface or LogController where all log functions are used

File: server/routes/log.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+12/10/2016, 4:01:43 PM: User, UPDATE, Schedule Start Time: 10, (run name: O_2_D_CTI, audit id: 289201545)
+12/10/2016, 4:02:46 PM: User, UPDATE, Status Code: 1, (run name: O_2_D_CTI, audit id: 289201542)
```
Comment (ProjectZed) [inline server/routes/log.txt]: Clean these files, there is a log.txt in server and in server/routes, which one are we actually using?

File: server/routes/writeLog.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
+module.exports = {
```
Comment (ProjectZed) [inline server/routes/writeLog.js]: why does this js file exist again? Just like the log files how there are two, which one are we using?

**Comment by nshoang0808** [comment]
I made changes according to your requirement. Please check it

**Pull request merged**